partner_contact_age_range/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2019-Today: Druidoo (<https://www.druidoo.io>)
```

Comment:
Please do not put copyright lines in __init__ files.

Reply:
@NL66278 : remove copyright lines from init files